app/wfbngrtl8812/src/main/cpp/WfbngLink.cpp

```diff
@@ -117,6 +117,7 @@ int WfbngLink::run(JNIEnv *env, jobject context, jint wifiChannel, jint bw, jint
 
     rtl_devices.emplace(fd, wifi_driver->CreateRtlDevice(dev_handle));
     if (!rtl_devices.at(fd)) {
+        libusb_exit(ctx);
```

Comment:
do we really want to deinit libusb here?
theoretically we could have a faulty device but another could function fine and we want to try and continue executing the application

Reply:
ctx here is a local variable. If we don't deinit it, it would be lost upon return.